.github/workflows/main.yml

```
@@ -0,0 +1,52 @@
+name: main
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+jobs:
+  build-and-test:
+    strategy:
+      matrix:
+        node:
+          - '10'
+          - '12'
+          - '14'
+    runs-on: ubuntu-latest
+    name: Tests on node ${{ matrix.node }}
```

Just a small suggestion:
```suggestion
    name: Tests on Node.js v${{ matrix.node }}
```